Library/Homebrew/dev-cmd/vendor-gems.rb

```diff
@@ -30,7 +30,7 @@ def vendor_gems
 
     Homebrew.install_bundler!
 
-    ENV["BUNDLE_WITH"] = "sorbet"
+    ENV["BUNDLE_WITH"] = VALID_GEM_GROUPS.join(":")
```

Comment:
Similarly: I think it'd be good that changes to this file always trigger the `vendor-gems` workflow. Perhaps even the `Gemfile` changing or `utils/gems` etc. should trigger this workflow too? Not really any harm in running it and would be good to catch at least the cases we had before where this only noticed to be failing after the fact.

Reply:
iirc it won't work on fork PRs as is but we could try add a "read-only" mode.

(same with Sorbet workflow below)

Reply:
yeh, it won't commit to them but the failures we've seen have always occurred before committing so I think the "do it but don't push the result" flow would be worth having 👍🏻.

Reply:
Done in #15185 so I don't have to recreate this on a non-fork (but I'll rebase this on that before merging).